File: app/src/main/java/dev/leonlatsch/olivia/database/model/MessageType.java
package dev.leonlatsch.olivia.database.model;

import com.fasterxml.jackson.annotation.JsonProperty;

public enum MessageType {
    @JsonProperty("UNDEFINED")
    UNDEFINED,

    @JsonProperty("TEXT")
    TEXT,

    @JsonProperty("IMAGE")
    IMAGE,

    @JsonProperty("AUDIO")
    AUDIO,

    @JsonProperty("VIDEO")
    VIDEO
}

File: app/src/main/kotlin/dev/leonlatsch/kolibri/database/model/MessageType.kt
package dev.leonlatsch.kolibri.database.model

import com.fasterxml.jackson.annotation.JsonProperty

enum class MessageType {
    @JsonProperty("UNDEFINED")
    UNDEFINED,

    @JsonProperty("TEXT")
    TEXT,

    @JsonProperty("IMAGE")
    IMAGE,

    @JsonProperty("AUDIO")
    AUDIO,

    @JsonProperty("VIDEO")
    VIDEO
}
